### DISCUSSION TITLE
Erro

### DISCUSSION THREAD

**Discussion by gmadeira** [Flutterando/forum#121] [Discussões gerais]
Boa tarde, estava rodando o progra tudo legal igual ao seu, de repente pediu flutter update e começou a dar esta mensagem outravez
 'WHERE' não é reconhecido como um comando interno
ou externo, um programa operável ou um arquivo em lotes.
'WHERE' não é reconhecido como um comando interno
ou externo, um programa operável ou um arquivo em lotes.
Error: PowerShell executable not found.
       Either pwsh.exe or PowerShell.exe must be in your PATH.

**Comment by EuHigorBarbosa** [comment]
Oi GMadeira, tudo bem? 
Olha, você descreveu poucas informações sobre o problema que você está passando. 
Mas ao que tudo indica, se você adicionar esse programa (PowerShell.exe) no seu path, então você vai conseguir utilizar o comando dele no prompt de comando. 

Da uma pesquisada na internet sobre como adicionar programas ao path do windows. 
Os programas que são adicionados no path podem ser acessados no prompt de comando.
